Seeder/source/models.py

```diff
@@ -387,9 +388,9 @@ def stripped_main_url(self):
         """
         url = self.url
         if url.startswith("http://"):
-            return url.lstrip("http://")
+            return url[7:]
         elif url.startswith("https://"):
-            return url.lstrip("https://")
+            return url[8:]
```

Comment:
to je divny, ze to nefungovalo

```
In [1]: "https://google.com".lstrip("https://")
Out[1]: 'google.com'


```

Reply:
Jj, funguje to na vetsine url v pohode, ale zaroven to vymaze ty znaky "https:/" i za lomitkama, kdyz tam nejake jsou, takze kdybys mel neco jako `"https://httpsdirectory.com".lstrip("https://")` tak by to vratilo jenom "directory.com"

Reply:
huh, wtf